HomeWork_Finish.py

```diff
@@ -0,0 +1,57 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+import time
+
+def setup_driver():
```

Comment:
це має бути фіктура. 
```suggestion
@pytest.fixture
def setup_driver():
```